src/Uno.UI/UI/Xaml/UIElement.skia.cs

```diff
@@ -94,7 +96,9 @@ internal ContainerVisual Visual
 				if (_visual == null)
 				{
 					_visual = Window.Current.Compositor.CreateContainerVisual();
+#if ENABLE_CONTAINER_VISUAL_TRACKING
 					_visual.Comment = $"Owner:{GetType()}/{(this as FrameworkElement)?.Name}";
+#endif
```

Comment:
@Youssef1313 I think we should start using more linker substitutions for that (that particular feature being disabled by default)